test/e2e/data/e2e_conf.yaml

```diff
@@ -71,6 +71,17 @@ providers:
             new: "imagePullPolicy: IfNotPresent"
           - old: --metrics-bind-addr=127.0.0.1:8080
             new: --metrics-bind-addr=:8080
+      - name: v1.1.99 # next;
+        value: "https://storage.googleapis.com/artifacts.k8s-staging-cluster-api.appspot.com/components/nightly_main_20211119/core-components.yaml"
```

Comment:
NOTE: Those versions have to be upgraded to at least `nightly_main_20211120` before merge to include the bugfix in https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/2939